packages/Tabs/src/components/Tab/Tab.js

```diff
@@ -37,6 +37,14 @@ export default function Tab(props) {
   const handleClick = isDisabled ? () => {} : onClick;
   const tabIndex = isSelected ? 0 : -1;
 
+  if (href && isDisabled) {
```

Comment:
Instead if the following condition was just modified as `if (href && !isDisabled)` you could eliminate this whole block and let it render the `<RawButton>` below... no?